hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HStore.java

```diff
@@ -1120,9 +1120,25 @@ public StoreFileWriter createWriterInTmp(long maxKeyCount, Compression.Algorithm
     if (isCompaction) {
       // Don't cache data on write on compactions, unless specifically configured to do so
       writerCacheConf = new CacheConfig(cacheConf);
-      writerCacheConf.setCacheDataOnWrite(cacheConf.shouldCacheCompactedBlocksOnWrite());
+      final boolean shouldCacheCompactedBlocksOnWrite = cacheConf
```

Comment:
Sorry for being late here.
The var name here sounds like a method name. Ideally it should be a noun. 'cacheCompactedBlocksOnWrite'